collections/hub/golden_path/feature_tests/duplicate_handling/transfers/original_transfer_at_committed.json

```diff
@@ -731,7 +731,7 @@
                 "id": 17,
                 "description": "Callback Header - content-type is application/vnd.interoperability.quotes+json;version=1.0",
                 "exec": [
-                  "expect(callback.headers['content-type']).to.equal('application/vnd.interoperability.transfers+json;version=1.0')",
+                  "expect(callback.headers['content-type']).to.have.string('application/vnd.interoperability.transfers+json;version=1')",
```

Comment:
why not `expectedTransfersVersion` here? and use of .equal assertion, since its a callback and the sim versions are set for QA

Reply:
This is for the second duplicate transfer and the callback is coming from switch in this case. The switch is not sending the same content-type of the original callback.

Reply:
yes, so we need two sets of versions - switch resource versions and simulator resource versions which can be set, that way we know what to test for, for sure.. that possibly increases the scope on this quite a bit, but that was my initial proposal... to set and use both Switch and simulator versions separately

Reply:
As we discussed over the call, I will create a followup story for that.